AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestResultDownloader.kt

```diff
@@ -66,7 +67,7 @@ internal class TestResultDownloader(
                             target = downloadFolder,
                             filter = { name ->
                                 // these are logs per test, they are plenty in numbers so lets not download them
-                                !name.contains("test_cases")
+                                includeTestCaseArtifacts || !name.contains("test_cases")
```

Comment:
this will increase the numbers quite a lot. (log file per test, we have 50-60K of them)

i need to see the androidx.dev side but i was thinking that we can make androidx.dev pass everything we need (FTL Test matrix) and we can return it the log file without re-discovering all of it in TestExecutionStore.

Reply:
Right, I am not really downloading all logs at once on androidx.dev side. I just added this option for later if we want to do bulk upload of artifacts. (As of now, I have no idea how to do bulk upload for artifacts).

Reply:
we should only upload logcat for tests that have failed. There is no value in logcat for passing methods.